### PR TITLE
feat(cardano-services): adds log of incoming requests

### PR DESCRIPTION
**Pull request by iccicci**
# Context

We need to increase the verbosity of our log

# Proposed Solution

Now incoming requests are logged.

### CODE REVIEW

File: packages/cardano-services/src/Http/HttpServer.ts [hunk 1]
```diff
@@ -93,6 +93,15 @@ export class HttpServer extends RunnableModule {
       );
     }
 
+    const requestLogger = contextLogger(this.logger, 'request');
+
+    this.app.use((req, _res, next) => {
+      const { body, method, path, query } = req;
+
+      requestLogger.debug({ body, method, path, query, time: Date.now() });
```
Comment (mirceahasegan) [inline packages/cardano-services/src/Http/HttpServer.ts:101]: I am not sure if the `time` part is needed. I think it should/is added by the logger implementation (by bunyan for example).

**Review by mkazlauskas — APPROVED** [review]
LGTM

File: packages/cardano-services/src/Http/HttpServer.ts (hunk 1 above)
Comment (iccicci) [reply packages/cardano-services/src/Http/HttpServer.ts]: Yes... loggers _usually_ add time by themselves...
Removed in 60c13c5fa

**Review by mirceahasegan — DISMISSED** [review]

**Review by rhyslbw — DISMISSED** [review]
Ready for squashing

**Review by rhyslbw — APPROVED** [review]

**Review by mirceahasegan — APPROVED** [review]

**Pull request merged**